particles/Tutorial/Kotlin/5_Collections/Collections.arcs

```diff
@@ -1,9 +1,7 @@
 // Tutorial: Collections
 // Shows how to use an input list, and store data in a manifest file.
 
-schema PersonDetails
-  name: Text
-  age: Number
+import './CollectionsSchemas.arcs'
```

Comment:
I've updated tutorials 4-6. I'm not so sure about updating TTT. Extracting the schemas out of the game seems like a half measure; if we refactor that, we should go ahead and break out the game into several manifest files. Since that is a more involved effort, I argue it should wait for a future PR.